packages/visual-editing/src/ui/ElementOverlay.tsx

```diff
@@ -160,8 +164,15 @@ const ElementOverlayInner: FunctionComponent<ElementOverlayProps> = (props) => {
 
       {title && (
         <Tab gap={1} paddingBottom={1}>
-          <Labels gap={2} padding={2}>
-            <Text size={1}>{Icon}</Text>
+          <Labels gap={3} padding={3}>
```

Comment:
Were these sizing adjustments discussed with anyone else? The spacing here now differs from the `Open in Studio` link. Happy to consider but we should probably align.

Reply:
Good shout! These UI updates are quite rough. I am planning on meeting with Marius to discuss a few ideas for taking it further/polishing. I'll make sure the 2 contexts align 👀

Reply:
Sounds good! Shall we maybe revert the spacing for now, get this PR merged and then handle those on a separate branch?

Reply:
Sounds good! Done :)